src/service/resources-impl.js

```diff
@@ -1344,11 +1344,13 @@ export class ResourcesImpl {
       return;
     }
 
+    const lastDequeueTime = this.exec_.getLastDequeueTime();
```

Comment:
Could also fix this by having the pqueue init lastDequeueTime to `Infinity`. Which is weird. But so is the value "0" for meaning "never dequeued yet".

Reply:
Yea, it probably should be initialized to NaN.